netbox/utilities/testing/api.py

```diff
@@ -460,6 +462,8 @@ def _build_query(self, name, **filters):
                     # TODO: Come up with something more elegant
                     # Temporary hack to support automated testing of reverse generic relations
                     fields_string += f'{field_name} {{ id }}\n'
+                elif inspect.isclass(field.type) and issubclass(field.type, IPAddressFamilyType):
```

Comment:
Not really happy about this one here, if there's a better way of doing this I'm happy to change it.